### PR TITLE
Update cache-http.sh

### PR DESCRIPTION
**Pull request by jbuettnerbild**
If the target folder changes, the name of the tar file must also change.

### CODE REVIEW

File: cache-http.sh [hunk 1]
```diff
@@ -15,10 +15,18 @@ if [ -z $INPUT_LOCK_FILE ]; then
     exit;
 fi
 
+echo "check connection"
+curl \
+    -u $INPUT_BASIC_AUTH_USERNAME:$INPUT_BASIC_AUTH_PASSWORD \
+    -X GET \
+    -x "$INPUT_HTTP_PROXY" \
+    $INPUT_CACHE_HTTP_API/health \
```
Comment (kevincobain2000) [inline cache-http.sh]:
```suggestion
    $INPUT_CACHE_HTTP_API/health
```

File: cache-http.sh [hunk 2]
```diff
@@ -15,10 +15,18 @@ if [ -z $INPUT_LOCK_FILE ]; then
...
+
 shaLockfile=`openssl sha1 $INPUT_LOCK_FILE |awk '{print $2}'`
 shaInstallCommand=`echo $INPUT_INSTALL_COMMAND|openssl sha1|awk '{print $2}'`
+shaDestinationFolder=`echo $INPUT_DESTINATION_FOLDER|openssl sha1|awk '{print $2}'`
 
-tarFile=$RUNNER_OS-$INPUT_VERSION-$shaInstallCommand-$shaLockfile.tar.gz
+tarFile=$RUNNER_OS-$INPUT_VERSION-$shaInstallCommand-$shaLockfile-$shaDestinationFolder.tar.gz
```
Comment (kevincobain2000) [inline cache-http.sh:29]: I don't think destination folder has any effect on the dependency caching. It is the folder where the tar will be untarred to.

Comment (jbuettnerbild) [reply cache-http.sh:29]: In my example, in addition to the node_modules folder, I have a second one in which I also have artifacts that I want to cache. Both folders will be tar and untarred. However, I did add the second folder until the second Workflow run, so there was already a tar in the cache that contained only one folder. Is certainly just an edgecase. I could have thrown away the cache, too.

Comment (kevincobain2000) [reply cache-http.sh:29]: Thanks for the clarification. I have pushed the tag v2.1 https://github.com/kevincobain2000/action-cache-http/releases/tag/v2.1

**Comment by jbuettnerbild** [comment]
Bildschirmfoto 2021-08-23 um 09 39 25

**Comment by jbuettnerbild** [comment]
apparently i accidentally changed the rights to the script

**Comment by jbuettnerbild** [comment]
100755 → 100644

**Comment by kevincobain2000** [comment]
will fix

**Comment by kevincobain2000** [comment]
done https://github.com/kevincobain2000/action-cache-http/commit/fcb822b12d3ac4a77c431b495ce2c98838ed61e4

**Comment by jbuettnerbild** [comment]
thx

**Pull request merged**